src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -95,6 +95,18 @@ public static int Run(string[] args)
         {
             DebugHelper.HandleDebugSwitch(ref args);
 
+            // Fix for https://github.com/Microsoft/vstest/issues/1453
+            // Try to run dll/exe directly using the VSTestForwardingApp
+            if (ContainsBuiltTestSources(args))
+            {
+                var convertedArgs = new VSTestArgumentConverter().Convert(args, out var ignoredArgs);
+                if (ignoredArgs.Any())
+                {
+                    Console.WriteLine(LocalizableStrings.IgnoredArgumentsMessage, string.Join(" ", ignoredArgs));
```

Comment:
@livarcocc Is there a better way to log a warning to the user?

Reply:
We usually write go though "Reporter" class like this 

https://github.com/dotnet/cli/blob/95c6eff6daa1a69f29c42b2d405400ad44bdec91/src/dotnet/commands/dotnet-sln/list/Program.cs#L35

We don't have special thing on warning. We have a "output string".Yellow(), helper to make it yellow, but that's all